lib/tasks/rails_db_views_tasks.rake

```diff
@@ -25,28 +25,17 @@ namespace :db do
 
     creator.drop_views
   end
+end
 
-  task :migrate => :environment do
-    Rake::Task['db:create_views'].invoke
-  end
-
-  task :rollback => :environment do
-    Rake::Task['db:create_views'].invoke
-  end
+# prepend "db" namespace
```

Comment:
While I agree that your refactor is cleaner, I think we should touch as little of the existing code as possible as this is a fork (and if we keep this gem we should work to get these changes back into the main branch).  Can you just add the code to match the Gem's style?